reference/6/Microsoft.PowerShell.Utility/Invoke-RestMethod.md

```diff
@@ -249,6 +265,23 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -FollowRelLink
+Indicates the cmdlet should follow relation links.
+
+To set how many times to follow relation links, use the `-MaximumFollowRelLink` parameter.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
```

Comment:
Should be Int32.MaxValue (or however we represent it in the docs)

Reply:
This is the switch parameter, the default shouldn't be Int32.MaxValue.

Reply:
Sorry, I misread that this section was for `MaximumFollowRelLink`.  All good.